src/main/scala/com/advancedtelematic/campaigner/db/Campaigns.scala

```diff
@@ -233,7 +233,7 @@ protected [db] class CampaignStatusTransition(implicit db: Database, ec: Executi
     val countFinishedDevices =
```

Comment:
As @houcros [mentioned](https://github.com/advancedtelematic/campaigner/pull/101/files/5c6f04bfeccb7bc5e0c65d216dde873041e1c590#r2598533290):

Something is odd. If we make this modification, then `val countFinishedDevices` is misleading, because "finished" is defined as "either successful or failed", as in the method `countFinished` up in this file. I feel either this modification is not needed or the name should be changed, but I'm not sure...

Reply:
I think the modification makes sense, but the name becomes misleading, that's true. I will also rename other variables to reflect better whether a number is coming from a group stats or device updates table.